CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [19.1.2] - 2022-03-23
+### Changed
+- Text attachments in the html report are now displayed as collapsible sections
```

Comment:
I think it's worth adding an entry under "Added" that we now display filenames from attachments.

Reply:
done